examples/serviceList/serviceList.py

```diff
@@ -0,0 +1,66 @@
+# -*- encoding: utf-8 -*-
+
+import datetime
+from tabulate import tabulate
+import ovh
+
+# Services type desired to mine. To speed up the script, delete service type you don't use!
+service_types 	= [ 
+	'allDom',
+	'cdn/dedicated',
+	'cdn/website',
+	'cdn/webstorage',
+	'cloud/project',
+	'cluster/hadoop',
+	'dedicated/housing',
+	'dedicated/nas',
+	'dedicated/nasha',
+	'dedicated/server',
+	'dedicatedCloud',
+	'domain/zone',
+	'email/domain',
+	'email/exchange',
+	'freefax',
+	'hosting/privateDatabase',
+	'hosting/web',
+	'hosting/windows',
+	'hpcspot',
+	'license/cloudLinux',
+	'license/cpanel',
+	'license/directadmin',
+	'license/office',
+	'license/plesk',
+	'license/sqlserver',
+	'license/virtuozzo',
+	'license/windows',
+	'license/worklight',
+	'overTheBox',
+	'pack/xdsl',
+	'partner',
+	'router',
+	'sms',
+	'telephony',
+	'telephony/spare',
+	'veeamCloudConnect',
+	'vps',
+	'xdsl',
+	'xdsl/spare',
+]
+
+# Create a client using ovh.conf
+client = ovh.Client()
+
+services_will_expired = []
+
+# Check all OVH product (service type)
+for service_type in service_types:
+	service_list = client.get('/%s' % service_type )
+
+	# If we found you have this one or more of this product, we get these informations
```

Comment:
"For each service, if any, get it's expiration date and status from the ``serviceInfos`` API"